packages/langgraph-agent/src/scripts/loadVectorStore.ts

```diff
@@ -0,0 +1,31 @@
+import { loadPdfsFromDirectory } from "../azure/find_pdfs.js";
+import { updateEnv } from "../utils/update_env.js";
+import { getEmbeddingClient } from "../azure/embeddings.js";
+import { fileURLToPath } from "url";
+import path from "path";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+const NORTHWIND_PDF_DIRECTORY = "../data";
+
+async function loadData(embeddings: any): Promise<void> {
```

Comment:
You can import `EmbeddingsInterface` from `"@langchain/core/embeddings"` instead of any